--- sope-SOPE-4.0.2/packaging/rhel/sope.spec.orig	2018-10-10 09:59:39.286077117 -0700
+++ sope-SOPE-4.0.2/packaging/rhel/sope.spec	2018-08-24 06:03:11.000000000 -0700
@@ -1,7 +1,7 @@
 %define apache_modules_dir %{_usr}/lib/httpd/modules
 %define apache_conf_dir    %{_sysconfdir}/httpd/conf.d
-%define oracle_support     1
-%{?el7:%define oracle_support 1}
+%define oracle_support     0
+%{?el7:%define oracle_support 0}
 
 Summary:      SOPE
 Name:         sope%{sope_major_version}%{sope_minor_version}
@@ -16,7 +16,7 @@
 Source:       %{sope_source}
 Prefix:       /usr
 BuildRoot:    %{_tmppath}/%{name}-%{version}-%{release}-root
-BuildPreReq:  gnustep-make gcc-objc postgresql-devel
+BuildRequires: gnustep-make gcc-objc postgresql-devel openldap-devel gnustep-base-devel gnustep-base libxml2-devel gmp-devel avahi-devel gnutls-devel libgcrypt-devel libffi-devel libxslt-devel libicu-devel python
 
 %{?el5:BuildRequires: mysql-devel}
 %{?el6:BuildRequires: mysql-devel}
@@ -287,7 +287,7 @@
 
 %prep
 rm -fr ${RPM_BUILD_ROOT}
-%setup -q -n SOPE
+%setup -q -n sope-SOPE-%{sogo_version}
 
 # ****************************** build ********************************
 %build
